wpilibNewCommands/src/test/native/cpp/frc2/command/CommandScheduleTest.cpp

```diff
@@ -95,6 +101,92 @@ TEST_F(CommandScheduleTest, SchedulerCancel) {
   EXPECT_FALSE(scheduler.IsScheduled(&command));
 }
 
+TEST_F(CommandScheduleTest, CancelNextCommand) {
+  CommandScheduler scheduler = GetScheduler();
+
+  frc2::RunCommand* command1PtrPtr = nullptr;
+  frc2::RunCommand* command2PtrPtr = nullptr;
```

Comment:
```suggestion
  frc2::RunCommand* command1Ptr = nullptr;
  frc2::RunCommand* command2Ptr = nullptr;
```
These are no longer pointer to pointer.

Reply:
Oops... forgot to change them back after fighting with C++ lambdas

## Added in 93ad7b8